ETC_en.tsv

```diff
@@ -5200,24 +5200,24 @@ ETC_20150317_005199	Don't come closer!
 ETC_20150317_005200	Please defeat the aggressive Harpia!
 ETC_20150317_005201	Please defeat Reaverpede!
 ETC_20150317_005202	I will not go easy on you!
-ETC_20150317_005203	Please defeat Rogue Master!
+ETC_20150317_005203	Please defeat the Rogue Master!
 ETC_20150317_005204	Here I go!
 ETC_20150317_005205	Please defeat Rocktortuga and obtain the natural agate stone!
-ETC_20150317_005206	Please defeat Schwarzer Reiter!
+ETC_20150317_005206	Please defeat the Schwarzer Reiter!
 ETC_20150317_005207	Please defeat Deadbone!
-ETC_20150317_005208	Please defeat Rajatoad!
+ETC_20150317_005208	Please defeat the Rajatoad!
```

Comment:
There is a Rajatoad card (so boss name), but no Rogue Master or Schwarzer Reiter